offgridplanner/optimization/processing.py

```diff
@@ -234,30 +244,70 @@ def collect_supply_opt_json_data(self):
             "energy_system_design": energy_system_design,
         }
 
+        response = requests.get("{SIM_API_HOST}/supply_schema/input", timeout=10)
```

Comment:
If the server has a timeout, I would issue a warning message and still proceed with the simulation request